src/main/resources/application.yml

```diff
@@ -23,6 +23,34 @@ spring:
         provider:
           hmpps-auth:
             token-uri: ${hmpps-auth.url}/oauth/token
+  jpa:
+    open-in-view: false
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
```

Comment:
this doesn't produce an error on start up does it? 

We get a warning in the logs for CVL around not having to specify the dialect

Reply:
yes, I can see it,
HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default) 

I'll remove in my next pr